### PR TITLE
🔧 Terms & Conditions on login

### PR DESCRIPTION
**Pull request by davidreneuw**
# Pull Request

## Description

Adds the terms and conditions checkbox on login page as well.

## Changes

- T&C checkbox on login
- Fix some of the wording (removing the word "trial")
- Remove localization duplicates

## Testing

- Existing tests pass

## Checklist

- [x] Code follows dotnet coding standards
- [x] Tests added/updated to cover changes

### CODE REVIEW

File: Portal/src/Datahub.Portal/i18n/localization.fr.json [hunk 1]
```diff
@@ -2972,17 +2972,5 @@
   "{0} credits consumed yesterday": "{0} crédits consommés hier",
   "{0} is already added to your workspace": "{0} est déjà ajouté à votre espace de travail",
   "{0} remaining credits of {1}": "{0} crédits restant de {1}",
-  "{0} successfully added to your workspace": "{0} ajouté avec succès à votre espace de travail",
-  "Changes to workspace have been saved": "Les modifications apportées à l'espace de travail ont été enregistrées",
```
Comment (ErikApption) [inline Portal/src/Datahub.Portal/i18n/localization.fr.json:2978]: Are you sure these entries are not used?

**Review by Sean-Stilwell — DISMISSED** [review]
Nice work!

image

Small i18n issue with this message, not sure where it comes from though. I encountered it by putting in my email then clicking Login immediately.

Alternatively you could disable the Login button while the checkbox is unchecked

**Review by ErikApption — CHANGES_REQUESTED** [review]

File: Portal/src/Datahub.Portal/i18n/localization.fr.json (hunk 1 above)
Comment (davidreneuw) [reply Portal/src/Datahub.Portal/i18n/localization.fr.json:2978]: Those were all marked as duplicates. For example, "Changes to workspace have been saved" is on line 451, "Queued Terraform update for workspace changes" is on line 2063, etc etc

**Review by ErikApption — DISMISSED** [review]

**Comment by Sean-Stilwell** [comment]
Just noticed this, could you swap "modalités et conditions" to "conditions générales"? We use that translation everywhere else so would be good to be consistent

**Review by ErikApption — APPROVED** [review]

**Pull request merged**